src/Deputy/Subscriber/ChangeAddressSubscriber.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+namespace AppBundle\Deputy\Subscriber;
+
+use AppBundle\Membership\AdherentEvent;
+use AppBundle\Membership\AdherentEvents;
+use AppBundle\Repository\DistrictRepository;
+use Doctrine\ORM\EntityManagerInterface;
+use Symfony\Component\EventDispatcher\EventSubscriberInterface;
+
+class ChangeAddressSubscriber implements EventSubscriberInterface
+{
+    private $districtRepository;
+    private $em;
+
+    public function __construct(DistrictRepository $districtRepository, EntityManagerInterface $em)
+    {
+        $this->districtRepository = $districtRepository;
+        $this->em = $em;
+    }
+
+    public function updateReferentTagWithDistrict(AdherentEvent $event): void
+    {
+        $adherent = $event->getAdherent();
+        $referentTag = $this->districtRepository->findReferentTagForDistrictByCoordinates($adherent->getLatitude(), $adherent->getLongitude());
+
+        if ($referentTag) {
+            $adherent->addReferentTag($referentTag);
```

Comment:
how do you check if the coordinates were changed?
`PROFILE_UPDATED` can be dispatched on simple first name modification, without changing the addresse and the coordinates.

Reply:
A check of the presence of a tag in the adherent referent tag list has been added.